spec/features/tslr_claim_journey_with_teacher_id_still_teaching_spec.rb

```diff
@@ -36,9 +36,13 @@
   end
 
   scenario "Selects school" do
-    # - "Selects suggested school retrieved from TPS" do
     navigate_to_still_teaching_page
 
+    # - Tries to continue without selecting option
+    click_on "Continue"
+    expect(page).to have_text("Select if you still work at #{eligible_school.name}, another school or no longer teach in England")
```

Comment:
This new test fails on master. The bug is that the current code looks up the school name for the error message from `eligibility.claim_school_name`. But for TID journey users it should be from `claim.recent_tps_school.name`.

Reply:
Now includes school name.